### PR TITLE
feat(filters): add built-in ExcludeFieldsMatchingPatternsFilter

### PR DESCRIPTION
**Pull request by theworkshopcom**

### CODE REVIEW

**Comment by theworkshopcom** [comment]
@fhussonnois we would like to contribute a few more filters into the repo, do you prefer a seperate PR per filter ? or all of them in one PR?

**Comment by fhussonnois** [comment]
Hi @theworkshopcom, thank you very much for your contribution. There's nothing wrong with having several filters in the same PR, as long as there's a separate commit. Is this PR ready for being merged ?

**Comment by theworkshopcom** [comment]
Thanks for confirming @fhussonnois , I will be adding 1 more filter and will comment back

**Comment by theworkshopcom** [comment]
hi @fhussonnois assuming no code review comments this should be ready to merge now

File: docs/content/en/docs/Developer Guide/filters.md
Comment (theworkshopcom) [inline docs/content/en/docs/Developer Guide/filters.md:1]: Do we need to manually update the `date` at the top of the file?

Comment (fhussonnois) [reply docs/content/en/docs/Developer Guide/filters.md:1]: Hi @theworkshopcom, Yes the date should be manually updated.

Comment (theworkshopcom) [reply docs/content/en/docs/Developer Guide/filters.md:1]: updated as well

**Pull request merged**